test/test_html_renderer.py

```diff
@@ -127,9 +128,9 @@ def test_thematic_break(self):
         self._test_token('ThematicBreak', '<hr />', children=False)
 
     def test_html_block(self):
-        content = output = '<h1>hello</h1>\n<p>this is\na paragraph</p>\n'
-        self._test_token('HTMLBlock', output,
-                         children=False, content=content)
+        output = '<h1>hello</h1>\n<p>this is\na paragraph</p>\n'
+        token = namedtuple('MockHtmlBlockToken', 'children')([namedtuple('MockRawTextToken', 'content')(output)])
```

Comment:
Clever use of `namedtuple()`. :) Maybe just the `Token` is redundant in the names, but that is a detail. ;)

Reply:
Thanks :) However, I've backed out this change to stick with the same pattern as the other test cases.